### PR TITLE
voltdb: Fix install block

### PR DESCRIPTION
**Pull request by dj-m**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Added %r around the regular expression in the install block & after-comma spacing

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Nice work @dj-m ✨

File: Formula/voltdb.rb [hunk 1]
```diff
@@ -17,11 +17,15 @@ class Voltdb < Formula
...
+    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader"],
+      %r{VOLTDB_LIB=\$VOLTDB_HOME\/lib}, "VOLTDB_LIB=$VOLTDB_HOME/lib/voltdb"
 
     (lib/"voltdb").install Dir["lib/*"]
     lib.install_symlink lib/"voltdb/python"
     prefix.install "bin", "tools", "voltdb", "version.txt", "doc"
   end
+
+  test do
+    assert_match version.to_s, shell_output("voltdb --version")
```
Comment (DomT4) [inline Formula/voltdb.rb]: Just FYI, in future, calling an executable that lives inside the Cellar should _always_ be done with a full path, i.e. `#{bin}/voltdb --version`

Comment (MikeMcQuaid) [inline Formula/voltdb.rb]: @DomT4 Missed that, will fix up.

Comment (DomT4) [inline Formula/voltdb.rb]: Beat ya to it 😃.

Comment (MikeMcQuaid) [inline Formula/voltdb.rb]: You already did in e934d03 I see 😀

**Comment by dj-m** [comment]
Should it be:

`system bin/"voltdb", "--version"`

in the test block?

**Comment by DomT4** [comment]
Heh, this is where it gets a bit silly. The preferred style for non-asserted tests is:
```ruby
system bin/"echo", "horse"
```
for example, but if you're asserting we prefer:
```ruby
assert_match "horse", shell_output("#{bin}/echo horse")
```
Or such. I wasn't too fussed about mixing & matching `#{bin}` and `bin` in the test blocks but ILZ has been on a kick against that so trying to save him stuff to fix later 😄.

**Comment by dj-m** [comment]
@MikeMcQuaid Thanks for the info! I know the string interpolation can be tricky... 🤔

File: Formula/voltdb.rb (hunk 1 above)
Comment (DomT4) [inline Formula/voltdb.rb]: Yeah, if I leave it it only sits on my mind until I do something about it 😅.

**Pull request merged**